doc/readthedocs-environment.yml

```diff
@@ -4,6 +4,7 @@ dependencies:
   - python==3.5
   - sphinx>=1.3.6
   - sphinx_rtd_theme
+  - sphinxcontrib-bibtex
```

Comment:
You should also add this to `doc/requirements.txt`.

Reply:
I added it, but I'm wondering why we have both files?
I thought `requirements.txt` is needed if we use pip and `readthedocs-environment.yml` if we use conda, which we are doing at the moment, aren't we?

Reply:
You are right, we are using `conda` on RTD and we don't need `requirements.txt` for that.
But it is still useful for users like me who want to install the necessary packages locally with

    python3 -m pip install -r doc/requirements.txt

like it's mentioned in http://python.sfstoolbox.org/en/latest/contributing.html

Reply:
That's true, and I'm using exactly this for building it locally.
Is there an advantage of using conda on RTD?

Reply:
The answer to this question keeps changing over time ...

In the past, there were problems with a broken `pandoc` installation on the default RTD deployment (without `conda`). And the versions of installed packages like e.g. `matplotlib` were hopelessly out-of-date (see #2).

Some issues that were solved by switching to `conda` but should (probably?) have been solved in the meantime:
https://github.com/spatialaudio/nbsphinx/commit/e10d0d8c7708607297f67e94bde505cd142d29f1#diff-c10bbd14be638b56c95217c4943387ff
https://github.com/spatialaudio/nbsphinx/issues/69
https://github.com/rtfd/readthedocs.org/issues/2521

The overall situation may have improved in the meantime.

I would actually prefer to switch away from `conda`, but in the past it just made less problems ...